src/t.jl

```diff
@@ -108,6 +153,15 @@ end
 testname(::EqualVarianceTTest) = "Two sample t-test (equal variance)"
 population_param_of_interest(x::TwoSampleTTest) = ("Mean difference", x.μ0, x.xbar) # parameter of interest: name, value under h0, point estimate
 
+"""
+    EqualVarianceTTest(x::AbstractVector{T<:Real}, y::AbstractVector{T<:Real})
+
+Perform a two-sample t-test of the null hypothesis that `x` and `y` come from a
+distributions with the same mean and equal variances against the alternative hypothesis
```

Comment:
"a distributions  with the same mean and equal variances" -> "distributions with equal means and variances". Same below.